### DISCUSSION TITLE
Thread and Matter protocol support

### DISCUSSION THREAD

**Discussion by girigupt** [edgexfoundry/community#124] [Q&A]
I am checking if there is any device-service being developed or available that supports Thread and Matters protocol?

**Comment by farshidtz** [comment]
There is currently none.
What are you trying to achieve with such integration? Use EdgeX in place of a Matter controller (commission and control) or plug EdgeX into an operational Matter system to interact with existing devices?

**Reply by girigupt** [reply]
Thanks @farshidtz , I had asked the question in response to a question I was asked. I would say, we should let edgex interact with existing devices in the Matter system & allow Matter devices to interact with non-matter devices/controllers via edgeX.
